src/data_source/storage/sql/queries/availability.rs

```diff
@@ -221,12 +221,12 @@ where
         // ORDER BY ASC ensures that if there are duplicate transactions, we return the first
         // one.
         let sql = format!(
-            "SELECT {BLOCK_COLUMNS}, t.index AS tx_index
+            "SELECT {BLOCK_COLUMNS}, t.idx AS tx_index
                 FROM header AS h
                 JOIN payload AS p ON h.height = p.height
-                JOIN transaction AS t ON t.block_height = h.height
+                JOIN transactions AS t ON t.block_height = h.height
                 WHERE t.hash = {hash_param}
-                ORDER BY (t.block_height, t.index) ASC
+                ORDER BY t.block_height ASC, t.idx ASC
```

Comment:
Nit: I think `ASC` is the default anyways so maybe we can just say `t.block_height, t.idx`?

Reply:
yeah I thought it is better for readability but I'll revert